build.yaml

```diff
@@ -762,12 +819,7 @@ job_type:
       # flocker.provision is responsible for creating the test nodes on
       # Rackspace, so we can actually run run-acceptance-tests from AWS
       on_nodes_with_labels: 'aws-ubuntu-trusty-T2Medium'
-      with_modules:
-        - flocker.acceptance.endtoend.test_dataset
-        - flocker.acceptance.integration.test_mongodb
-        - flocker.acceptance.integration.test_postgres
-        - flocker.acceptance.obsolete.test_cli
-        - flocker.acceptance.obsolete.test_containers
+      with_modules: *run_acceptance_modules
```

Comment:
they are, however I don't think they justify an expensive cycle of new JIRA ticket, new branch, commit, PR, code review request